src/client/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { addItem } from './actions';
+
+class SearchBar extends Component {
+  state = {
+    inputValue: '',
+    itemAddError: false
+  };
+
+  handleInputChange = e => {
+    this.setState({
+      inputValue: e.target.value
+    });
+  };
+
+  handleFormSubmit = e => {
+    e.preventDefault();
+    const { inputValue, itemAddError } = this.state;
+    const { addItem } = this.props;
+    const newItem = {
+      name: inputValue,
+      isOrdered: false
+    };
+
+    addItem(newItem).catch(err => {
+      this.setState({
+        itemAddError: true
+      });
+      console.error(err.message);
```

Comment:
remove console.error

Reply:
This error is handled by ui message in the next pull request.